app/controllers/v0/virtual_agent/virtual_agent_claim_controller.rb

```diff
@@ -8,7 +8,7 @@ module VirtualAgent
     class VirtualAgentClaimController < ApplicationController
       include IgnoreNotFound
       rescue_from 'EVSS::ErrorMiddleware::EVSSError', with: :service_exception_handler
-      if Settings.vsp_environment.present? && Settings.vsp_environment.downcase != 'development'
+      unless Settings.vsp_environment == 'localhost' || Settings.vsp_environment == 'development'
```

Comment:
Is the `.downcase` not needed here anymore?

Reply:
I don't think so, since `vsp_environment` is something we've set in the different `Settings.yml` files and I don't see any capitalized strings in any of those